tests/diffusion_dislocation/screen-output

```diff
@@ -13,16 +13,16 @@ Number of degrees of freedom: 1,237 (578+81+289+289)
    Solving temperature system... 0 iterations.
    Solving depleted_lithosphere system ... 0 iterations.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 16+0 iterations.
-      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.68721e-16, 1.05609e-16, 0.000715698
-      Relative nonlinear residual (total system) after nonlinear iteration 2: 0.000715698
+   Solving Stokes system... 21+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.68721e-16, 1.05609e-16, 0.000716824
+      Relative nonlinear residual (total system) after nonlinear iteration 2: 0.000716824
```

Comment:
It's a bummer that (at the moment) you need more iterations, but a good sign that the results are essentially identical.